source/tutorials/openfaas.rst

```diff
@@ -0,0 +1,363 @@
+.. _openfaas:
+
+OpenFaaS\*
+##########
+
+This tutorial shows how to set up OpenFaaS running on top of a Kubernetes\*
+cluster on |CL|, obtain |CL| based OpenFaaS templates, and develop an example
```

Comment:
What is the link to CL? Or it is just a short name for Clear Linux?

Reply:
Yes, it's a replacement variable that gets substituted with the full branding name. You'll have to build the docs locally to see that.